playground/miden-wasm/src/lib.rs

```diff
@@ -63,16 +63,61 @@ pub fn program(asm: &str, inputs_frontend: &str, output_count: u16) -> Vec<u64>
     )
     .map_err(|err| format!("Failed to generate exection trace = {:?}, and advice_tape = {:?}", err, advice_tape)).unwrap();
     
-    // TODO: Investigate why proof verification fails when outputCount > 1
-    //assert!(miden::verify(*program.hash(), &[], &outputs, proof).is_ok());
-    // tracked in https://github.com/0xPolygonMiden/examples/issues/19
     trace.program_outputs().stack_outputs(output_count as usize).to_vec()
 }
 
+/// Proves the program with the given inputs
+#[wasm_bindgen]
+pub fn prove_program(asm: &str, inputs_frontend: &str, output_count: u16) -> Vec<u64> {
+    console_error_panic_hook::set_once();
+    wasm_logger::init(wasm_logger::Config::default());
+
+    let assembler = Assembler::new();
+    let program = assembler.compile(&asm).expect("Could not compile source");
+    
+    let mut stack_init = <Vec<u64>>::new();
+    let mut advice_tape = <Vec<u64>>::new();
+
+    // TODO: We must catch all cases of inputs. But maybe in the frontend already.
```

Comment:
I don't understand this. Can you rephrase and also make sure it's tracked by an issue?

Reply:
This is tracked here - and the next PR that I will send to you https://github.com/0xPolygonMiden/examples/issues/18

I will rephrase